tools/c7n_left/pyproject.toml

```diff
@@ -23,7 +23,7 @@ c7n = {path = "../..", develop = true}
 
 click = ">=8.0"
 rich = "^13.1"
-tfparse = "^0.6"
+tfparse = "0.6.13"
```

Comment:
We can leave the `^0.6` constraint here - part of the c7n [release process](https://github.com/orgs/cloud-custodian/discussions/7509) involves a poetry update ([pkg-rebase](https://github.com/cloud-custodian/cloud-custodian/blob/242a5508a7defece72ad97d3c7a103b4a07a6492/Makefile#L86-L91)) that will update the locked tfparse version.

Separately, the release workflow uses the [freeze-wheel plugin](https://github.com/cloud-custodian/poetry-plugin-freeze) to be sure published package wheels have pinned versions for the full dependency graph.